### PR TITLE
refactor: upgrade accumulation to GP v0.5.2

### PR DESCRIPTION
**Pull request by danielvladco**
closes: #181

### CODE REVIEW

**Review by bamzedev — COMMENTED** [review]
Can we change [this](https://github.com/eigerco/strawberry/blob/main/internal/state/types.go#L46) now that it's fixed in the [GP](https://github.com/gavofyork/graypaper/compare/v0.4.5...v0.5.2#diff-abd897abe27e05cd5f28779d326efc03bf180542b29bc1a02d039a53a2996444R96)?

**Comment by danielvladco** [comment]
> Can we change [this](https://github.com/eigerco/strawberry/blob/main/internal/state/types.go#L46) now that it's fixed in the [GP](https://github.com/gavofyork/graypaper/compare/v0.4.5...v0.5.2#diff-abd897abe27e05cd5f28779d326efc03bf180542b29bc1a02d039a53a2996444R96)?

Done, thanks for catching this 👍

File: internal/state/types.go [hunk 1]
```diff
@@ -41,10 +41,10 @@ type AccumulationHistory [jamtime.TimeslotsPerEpoch]map[crypto.Hash]struct{} //
 
 // AccumulationState characterization of state components (equation 174 v0.4.5)
 type AccumulationState struct {
-	ServiceState       service.ServiceState                                                // Service accounts δ (d ∈ D⟨NS → A⟩)
-	ValidatorKeys      safrole.ValidatorsData                                              // Validator keys ι (i ∈ ⟦K⟧V)
-	WorkReportsQueue   [common.TotalNumberOfCores][PendingAuthorizersQueueSize]crypto.Hash // Queue of work-reports (q ∈ C⟦H⟧QHC)
-	PrivilegedServices service.PrivilegedServices                                          // Privileges state (x ∈ (NS, NS, NS, D⟨NS → NG⟩))
+	ServiceState             service.ServiceState                                                // Service accounts δ (d ∈ D⟨NS → A⟩)
+	ValidatorKeys            safrole.ValidatorsData                                              // Validator keys ι (i ∈ ⟦K⟧V)
+	PendingAuthorizersQueues [common.TotalNumberOfCores][PendingAuthorizersQueueSize]crypto.Hash // Queue of work-reports (q ∈ C⟦H⟧QHC)
```
Comment (bamzedev) [inline internal/state/types.go:46]: Forgot the comment.

**Review by bamzedev — COMMENTED** [review]
Left the comment

**Review by bamzedev — APPROVED** [review]

**Pull request merged**